main.go

```diff
@@ -56,7 +56,7 @@ func run() error {
 		return fmt.Errorf("failed to render template: %v", err)
 	}
 
-	fmt.Printf("::set-output name=result::%s", escape(output))
+	os.Setenv("GITHUB_OUTPUT", fmt.Sprintln(os.Getenv("GITHUB_OUTPUT"))+fmt.Sprintf("result=%s", escape(output)))
```

Comment:
If by `$GITHUB_OUTPUT` is null, we end up saving the env variable with extra line at the beginning, therefore had to do the following:

```suggestion
	if os.Getenv("GITHUB_OUTPUT") != "" {
		os.Setenv("GITHUB_OUTPUT", fmt.Sprintln(os.Getenv("GITHUB_OUTPUT"))+fmt.Sprintf("result=%s", escape(output)))
	} else {
		os.Setenv("GITHUB_OUTPUT", fmt.Sprintf("result=%s", escape(output)))
	}
```

Reply:
Tested it here https://go.dev/play/p/z60cVkxBX6K